en/mdn_plus_terms.md

```diff
@@ -21,10 +21,10 @@ Mozilla is a global organization and our computers and service providers’ comp
 
 __Your Use of the Service.__ Mozilla gives you permission to use the MDN Plus Service according to these Terms of Service. This permission is only for your personal use. You may not transfer, sublicense, resell, or commercially exploit the service.
 
-You give Mozilla all rights necessary to operate the Service, including processing the data you upload to or access through the Service including any notes you make or contributions to the website. This also includes sending you notifications and processing and storing your information such as bookmarks and labels. You can find more information about the data Mozilla processes in the [MDN Plus Privacy Notice](https://www.mozilla.org/privacy/mdn-plus/).
+You give Mozilla all rights necessary to operate the Service, including processing the data you upload to or access through the Service including any notes you make or contributions to the website. This also includes sending you notifications and processing and storing your information such as saved pages and labels. You can find more information about the data Mozilla processes in the [MDN Plus Privacy Notice](https://www.mozilla.org/privacy/mdn-plus/).
 You also agree that your use of the Service will comply with Mozilla’s [Conditions of Use](https://www.mozilla.org/about/legal/acceptable-use/). You are solely responsible for the content you transmit and the consequences.
 
-__Your Feedback and Suggestions.__ If you give Mozilla any ideas, suggestions, or feedback about Firefox Relay or the services you use with the Firefox Relay, you give Mozilla permission to use them for free and without any additional obligations.
+__Your Feedback and Suggestions.__ If you give Mozilla any ideas, suggestions, or feedback about MDN Plus or the services you use with the Firefox Relay, you give Mozilla permission to use them for free and without any additional obligations.
```

Comment:
@mafeld there is a second mention of Firefox Relay in this string. Is this intended?

Reply:
no. I'll fix that too. **sigh** thanks for catching.